contracts/mocks/OrderLibMock.sol

```diff
@@ -0,0 +1,69 @@
+pragma solidity ^0.4.23;
+
+import "../libraries/OrderLib.sol";
+
+contract OrderLibMock {
+  using OrderLib for OrderLib.OrderMappings;
+  using OrderLib for OrderLib.Order;
+
+  function _getQtyFilledOrCancelledFromOrder(
```

Comment:
@NFhbar thanks so much for the PR! Can you help me to understand your choice to go with a new style for function names here?  "_functionName()" vs "functionName()" which has been used elsewhere in the codes base?

Reply:
@pelsasser I usually mock functions by an underscore to indicate that are a mock functions instead of a function re declaration.  This is purely aesthetic and I can change it if you prefer.

Reply:
@NFhbar - thanks that makes a lot of sense. Maybe we can just stick a note somewhere in there on that style.

Reply:
Yeah, I'll add a top level comment indicating the purpose of the mock contract and the choice of style.